Tests/scripts/check_base_branch_for_external_pr.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import sys
+from Tests.test_utils import print_error, print_warning, print_color, LOG_COLORS
+
+
+def get_base_branch(pr_num):
+    """
+    Fetches the base branch name of PR num {pr_num}
+    :param pr_num: The PR number
+    :return: The name of the base branch if exists
+    """
+
+    # Disable insecure warnings
+    requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+    try:
+        url = 'https://api.github.com/repos/demisto/content/pulls/{}'.format(pr_num)
+        res = requests.get(url, verify=False)
+        if res.status_code != 200:
+            # If we didn't succeed to fetch the pr maybe it doesn't exist - then we don't want the build to fail
+            print_warning('Unable to fetch PR num {}'.format(pr_num))
```

Comment:
use f strings and return the api response as well

Reply:
all script must be in python 2.

Reply:
Why does it have to be python2? you are running it with py3..

Reply:
@ronykoz code isn't updated, changed to py3